nevergrad/benchmark/experiments.py

```diff
@@ -35,26 +35,25 @@
 
 
 @registry.register
-def mltuning(seed: tp.Optional[int] = None, overfitting: bool=False) -> tp.Iterator[Experiment]:
```

Comment:
@teytaud overfitting is not used, is that expected?

Reply:
If you update you should see it ? now it's used.

Reply:
naivemltuning uses it. It's the case in which the valid error (used in the optimization of hyperparameters) is also the error used in the evaluation function -- this is for distinguishing algorithms good at optimizing the valid loss from algorithms good at not overfitting.

Reply:
@teytaud no, I confirm after merging, the parameter is not used in `mltuning`, so `naivemltuning` is just `mltuning` again.
This is something `pylint` can help you catch, maybe we'll set up the commit hook some day ;)

Reply:
#669 adds a CI test that all variables are used, it's catching the `overfitting` variable. When you have fixed it I'll merge it and there will be less risk for this kind of issue ;)